src/app/auth/auth.guard.ts

```diff
@@ -1,19 +1,19 @@
 import { Injectable } from '@angular/core';
 import { ActivatedRouteSnapshot, CanActivate, RouterStateSnapshot, UrlTree } from '@angular/router';
 import { Observable } from 'rxjs';
-import { AuthInterceptorService } from '../auth-interceptor.service';
 import { Router } from '@angular/router';
+import { AuthService } from '../auth-service';
 
 @Injectable({
   providedIn: 'root'
 })
 export class AuthGuard implements CanActivate {
-  constructor(private route: Router) {}
+  constructor(private route: Router, private authService: AuthService) {}
   canActivate(
     route: ActivatedRouteSnapshot,
     state: RouterStateSnapshot): Observable<boolean | UrlTree> | Promise<boolean | UrlTree> | boolean | UrlTree {
-
-      if(!localStorage.getItem('token') == true) {
+      if(!(this.authService.isLoggedIn == true))
+      {
```

Comment:
we are not using any web storage ?? So on refresh the user will logout ??

Reply:
Added the localStorage to keep the user page on page refresh.

Reply:
localStorage from service